src/lightning_app/runners/multiprocess.py

```diff
@@ -63,6 +66,13 @@ def dispatch(self, *args: Any, on_before_run: Optional[Callable] = None, **kwarg
             storage_orchestrator.start()
 
             if self.start_server:
+                if check_port_already_used(self.port):
+                    logger.error(
+                        "A Lightning App is already running locally. "
+                        "HINT: Stop the previous App or run in the cloud with `--cloud`."
```

Comment:
Would maybe change this to:
```suggestion
                        "Stop the existing App or run in the cloud with `--cloud`."
```